proto/info/patch_envelope.proto

```diff
@@ -0,0 +1,45 @@
+// Copyright(c) 2023 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package org.lfedge.eve.info;
+option go_package = "github.com/lf-edge/eve/api/go/info";
+option java_package = "org.lfedge.eve.info";
+
+enum EvePatchEnvelopeState {
+    EVE_PATCH_ENVELOPE_STATE_UNKOWN = 0;
+    EVE_PATCH_ENVELOPE_STATE_ERROR = 1;
```

Comment:
Can you please annotate each of these states with a brief description above them as comment

Reply:
Done